comit/src/orderpool.rs

```diff
@@ -318,13 +318,21 @@ fn match_orders(
     };
 
     if left.swap_protocol != right.swap_protocol {
-        tracing::trace!("orders with different swap protocols don't match");
+        tracing::trace!(
+            "orders with different swap protocols don't match: {:?} {:?}",
+            left.swap_protocol,
+            right.swap_protocol
+        );
         return None;
     }
 
     // TODO: partial order matching
     if left.quantity != right.quantity {
-        tracing::trace!("orders with different quantities don't match");
+        tracing::trace!(
+            "orders with different quantities don't match: {:?} {:?}",
```

Comment:
Have you checked that this actually looks like in the logs? I think we should be good getting away with `Display` here. There is nothing that the `Debug` print gives us that couldn't also go into the `Display`.

Additionally, we could make the statement a bit nicer:

```suggestion
            "orders with different quantities don't match, {} != {}",
```